install/scripts/configure_os.sh

```diff
@@ -90,7 +90,9 @@ cp /opt/integralstor/integralstor/install/conf-files/db/*.db /opt/integralstor/i
 sqlite3 /opt/integralstor/integralstor/config/db/integralstor.db < /opt/integralstor/integralstor/install/conf-files/db/integralstor_db.schema
 
 # Populate cron entries
-(crontab -l 2> /dev/null; cat /opt/integralstor/integralstor/install/scripts/cron_entries.list) | crontab -
+# ALERT: clears existing entries
```

Comment:
If this is will be called for a generic non-fractalio install (possibly from an rpm install), then is it safe to remove all existing crontab entries that a user may already have?

Reply:
It is not a good idea to clear the existing entries. I considered the process of updates and upgrades: if we keep the existing cron entries, user specific and istor specific both may be present, and we will end up appending new Istor entries with the already existing Istor entries(from previous setup run), leaving us with duplicate entries.

Figuring out the previous entries of Istor programmatically was becoming odious because there may be entries from scheduled tasks and notifications. This requires version specific scripting. So I thought it's better to clear it until upgrading scripts are rolled out.